docs/apache-airflow/installation/upgrading.rst

```diff
@@ -57,6 +57,11 @@ when you choose to upgrade airflow via their UI.
 How to upgrade
 ==============
 
+Reinstall Apache Airflow™, specifying the desired new version. For example, in the case of a bootstrapped
+local instance, you can upgrade easily by setting the ``AIRFLOW_VERSION`` environment variable
```

Comment:
Hmm. That's still a little misleading. That works for quick-start (so the first doc linked below) - but when you go to "Installing airlfow" - second link  (and most of the times people are using PyPI to install) there is no `AIRFLOW_VERSION` variable mentioned whe installing from PyPI. So that might be a bit confusing for people.

Reply:
Thanks, I see how this would be confusing. I added guidance for upgrading a PyPI package.